src/main/java/com/adq/jenkins/xmljobtodsl/dsl/strategies/custom/DSLRenderParamsAsStringsStrategy.java

```diff
@@ -0,0 +1,46 @@
+package com.adq.jenkins.xmljobtodsl.dsl.strategies.custom;
+
+import com.adq.jenkins.xmljobtodsl.dsl.strategies.DSLMethodStrategy;
+import com.adq.jenkins.xmljobtodsl.parsers.PropertyDescriptor;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class DSLRenderParamsAsStringsStrategy extends DSLMethodStrategy {
```

Comment:
Do we need two separate classes for Params and for Choices?

Reply:
I forgot to push after I renamed this, but we needed a class for the params and a class for the method

Reply:
Otherwise, the default value was being rendered as a parameter. 
<img width="500" alt="Screenshot 2023-01-24 at 1 49 21 PM" src="https://user-images.githubusercontent.com/112515811/214382011-b0be975e-47ba-4575-a0d6-7ae0599baf04.png">